ocaml/xapi-consts/api_messages.ml

```diff
@@ -84,12 +84,24 @@ let vm_shutdown = addMessage "VM_SHUTDOWN" 5L
 
 let vm_rebooted = addMessage "VM_REBOOTED" 5L
 
+let vm_migrated = addMessage "VM_MIGRATED" 5L
+
+let vm_snapshoted = addMessage "VM_SNAPSHOTED" 5L
```

Comment:
This looks a little odd... I think it should be "snapshotted".

Reply:
Indeed, it's fixed!